bogoworld/main.py

```diff
@@ -0,0 +1,58 @@
+from random import randint
```

Comment:
Good work using the execution wrapper. For a script that is meant to be run directly, don't forget to include a shebang and make the file executable.

Since there are so many versions of python, it would help to leave a comment regarding which python version you are using!
```suggestion
#!/usr/bin/env python
# tested on python3.9
from random import randint
```